src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -261,6 +273,9 @@ private RouteFW resolveTarget(
         private final long routeId;
         private final long initialId;
         private final long replyId;
+        private final Map<String, MqttServerStream> subscribers;
+        private final Map<String, MqttServerStream> publishers;
+        private final Map<Integer, Subscription> subscriptionsByPacketId;
```

Comment:
Use `Int2ObjectHashMap<Subscription>` instead of `Map<Integer, Subscription>` to avoid auto-boxing.